.prettierrc

```diff
@@ -0,0 +1 @@
+{"singleQuote": true}
```

Comment:
Why was this added?

Reply:
It was to help with formatting. 
I'll add it to `.gitignore`